bee-hive/README.md

```diff
@@ -95,14 +95,24 @@ python run_workflow workflow.yaml
 
 * Configure environmental variables: `cp example.env .env`
 
-### Run workflow in local environment with container runtime (e.g. podman)
+### Run workflow in local environment with container runtime (e.g. podman) or kubernetes cluster
 
-* Run a local instance of the [Bee Stack](https://github.com/i-am-bee/bee-stack)
+* Prepare required environments
+  * Run a local instance of the [Bee Stack](https://github.com/i-am-bee/bee-stack)
+  * `curl` is required for `run` command
+  * for Kubernetes, `kubectl` and `sed` are required
+  * set CONTAINER_CMD, TARGET_IP and BUILD_FLAGS environment variables to adjust bee-hive.sh script
+    * CONTAINER_CMD: docker, podman. nerdctl, etc..
+    * TARGET_IP: 127.0.0.1:"NodePort of the bee-hive service" for kubernetes deployment
+    * BUILD_FLAGS: image build command extra flag (e.g. "--namespace k8s.io" for nerdctl)
+
+* Build bee-hive container image: `./bee-hive.sh build` in bee-hive/bee-hive directory
+  * The built bee-hive:latest image need to be pushed/placed in the local retistry for kubernetes deployment
 
-* Build bee-hive container image: `./bee-hive.sh build` in	bee-hive/bee-hive directory
+* Deploy bee-hive: `./bee-hive.sh deploy BEE_API=http://xxx.xxx.xxx.xxx:4000 BEE_API_KEY=sk-proj-testkey`
+  * To deploy to kubernetes cluster, user `deploy-k` instead `deploy` as the first argument
```

Comment:
nit: let's make two sections for the deploy, one for docker and one for kubernetes. that would also allow for including some kubernetes-specific instructions:
* to load the image into the cluster 
* to port-forward the bee-hive service, if necessary 

since both of those will differ depending on the k8s used (rancher, kind, etc.), I don't think we need to include specific instructions, but we should mention that the steps are needed

Reply:
@psschwei Thanks!  I'll create another PR for the readme update!

Reply:
feel free to merge when you're ready (or if you want to wait to see if anyone has comments that's cool too)